src/tools/kata-ctl/src/arch/x86_64/mod.rs

```diff
@@ -8,10 +8,53 @@ pub use arch_specific::*;
 
 mod arch_specific {
     use anyhow::Result;
+    use crate::check;
+
+    static PROC_CPUINFO: &str = "/proc/cpuinfo";
+    static CPUINFO_DELIMITER: &str = "\nprocessor";
+    static CPUINFO_FLAGS_TAG: &str = "flags";
+
+    const CPU_FLAGS_INTEL: &str = "vmx lm sse4_1";
```

Comment:
Of course this works, but it might be better to make it a slice. I won't lie, it'll look ugly - brace yourself... !

```rust
const FLAGS: &'static [&'static str] = &["lm", "sse4_1", "vmx"];
```